app/service/dictionary/find_service.rb

```diff
@@ -0,0 +1,24 @@
+module Dictionary
+  class FindService
+    def initialize(class_name, params)
+      @class_name = class_name
+      @params = params
+    end
+
+    def call
+      if params[:filter].present?
+        class_name.where('name LIKE ?', "%#{params[:filter]}%")
```

Comment:
better use ilike
and add pagination and default sorting)

Reply:
I see you have made Vlad s review changes. Good!